app/views/data_access/retrieval.html.erb

```diff
@@ -14,9 +14,9 @@
   <nav class="data-access-nav">
     <a href="<%=edsc_path "/data/configure?#{@retrieval.source}" %>"><i class="fa fa-arrow-circle-o-left"></i> Back to Data Access Options</a>
   </nav>
-  <header class="data-access-header">
-    <h1>Success!</h1>
-    <p>Your request has been processed.  See below for information on accessing your data.</p>
+  <header class="data-access-header" data-bind="visible: ui.projectList.collectionsToDownload().length == 0">
+    <h1>Request in Progress</h1>
+    <p>Your request is being processed.  Results will appear shortly.</p>
```

Comment:
Personally, I would remove the "Results will appear shortly" sentence. We don't know how long it will take in some cases. There is an effort underway by our underlying services to provide us with a time estimate, so that would be something we could tackle when that is in place.